torch/testing/_internal/logging_tensor.py

```diff
@@ -114,14 +120,19 @@ def log_input(name: str, var: object):
     logging.getLogger("LoggingTensor").info("input", (name,), {}, (var,))
 
 @contextlib.contextmanager
-def capture_logs() -> Iterator[List[str]]:
+def capture_logs(is_dispatch_mode=False) -> Iterator[List[str]]:
```

Comment:
To check my understanding, `is_dispatch_mode` switches on if we are using Logging Tensor, or if we are using the Mode, right?

If so, then nit: the name is a bit ambiguous, if `is_dispatch_mode=False` then sometimes "torch_function mode" comes into mind. Would just add a comment here for people coming in the future, or change this to `is_mode`